hack/azure-CVM-image-delete-job.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: azure-confidential-image-deletion
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 1
+  template:
+    metadata:
+      name: azure-confidential-image-deletion
+    spec:
+      securityContext:
+        runAsUser: 0 # needed for container mode dnf access
+
+      containers:
+      - name: delete
+        image: registry.access.redhat.com/ubi9/ubi:9.2
```

Even if that wouldn't change much here, it is good practice to restrict the extra privileges to the containers that need it :

```suggestion
      containers:
      - name: delete
        image: registry.access.redhat.com/ubi9/ubi:9.2
        securityContext:
          runAsUser: 0 # needed for container mode dnf access
```